src/main/java/jarvis/ui/MainWindow.java

```diff
@@ -0,0 +1,65 @@
+package jarvis.ui;
+
+import jarvis.logic.Jarvis;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+/**
+ * Controller for the main GUI.
+ */
+public class MainWindow extends AnchorPane {
+    @FXML
+    private ScrollPane scrollPane;
+    @FXML
+    private VBox dialogContainer;
+    @FXML
+    private TextField userInput;
+    @FXML
+    private Button sendButton;
+
+    private Jarvis jarvis;
+
+    private Image userImage = new Image(this.getClass().getResourceAsStream("/images/jarvis.jpg"));
+    private Image jarvisImage = new Image(this.getClass().getResourceAsStream("/images/jarvis.jpg"));
+
+    @FXML
+    public void initialize() {
+        scrollPane.vvalueProperty().bind(dialogContainer.heightProperty());
+        String logo =
+                "       _                  _     \n" +
+                        "      (_)___ _______   __(_)____\n" +
+                        "     / / __ `/ ___/ | / / / ___/\n" +
+                        "    / / /_/ / /   | |/ / (__  )\n" +
+                        " __/ /\\__,_/_/    |___/_/____/\n" +
```

Comment:
it might be a good idea to move the + operator to a new line when concatenating strings. This could help make the code easier to read and maintain. What do you think?